src/annotator/components/test/NotebookModal-test.js

```diff
@@ -14,12 +14,13 @@ describe('NotebookModal', () => {
 
   const outerSelector = '[data-testid="notebook-outer"]';
 
-  const createComponent = config => {
+  const createComponent = ({ attachTo, ...config } = {}) => {
     const component = mount(
       <NotebookModal
         eventBus={eventBus}
         config={{ notebookAppUrl: notebookURL, ...config }}
       />,
+      { attachTo },
```

Comment:
I'm not always attaching to the DOM, because it breaks other tests, and is relevant only to test the default focusing functionality.